content/blog/2015-05-24-custom-sd.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Plugging in your Custom Service Discovery
+created_at: 2015-05-24
+kind: article
+author: Fabian Reinartz
+---
+
+This week we released Prometheus v0.14 — a version with many long-awaited additions
+and improvements.
+
+On the user side, the the new means for service discovery aim to make it easier to
+dynamically change the targets Prometheus scrapes. [Consul](https://www.consul.io) is
+now supported out-of-the-box and a file-based plugin mechanism allows you to connect
+your own discovery to Prometheus.
+
+In this blog post, we will take a closer look at the latter option and provide a hands-on
+example to push your changes from etcd into Prometheus instantaniously.
+
+
+## How does it work?
```

Comment:
I think we should lead with Consul with an indication that we'll be adding many more out of the box.

Then go into doing it yourself as a fallback